pdns/dnsdistdist/populate

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 
 set -e
 set -x
```

Comment:
`set -e` and `set -x` are in POSIX http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html